tools.yml

```diff
@@ -1936,12 +2086,16 @@ tools:
     cores: 1
     mem: 12
   toolshed.g2.bx.psu.edu/repos/iuc/scanpy_cluster_reduce_dimension/scanpy_cluster_reduce_dimension/.*:
```

Comment:
pretty sure these 4 scanpy tools had their cores values removed for a reason too.